R/test-module.R

```diff
@@ -37,28 +42,28 @@
 #'   session$setInputs(x = 2)
 #'   stopifnot(myreactive() == 4)
 #'   stopifnot(output$txt == "I am 4")
-#' })
+#' }, !!multiplier_arg_name := 2, !!!more_args)
```

Comment:
I  think there should be two examples with `testModule`. The first one wouldn't use `!!` or `!!!` and simply pass `multiplier = 2`.

The next one could use `!!` and `!!!` (and there should be a comment saying this is for advanced users). Otherwise normal users will see this example and get confused or scared off, and by something that is only used in rare occasions.

Reply:
I tried to clean this up, let me know if it's what you had in mind.

Reply:
Looks good!